Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Base
+FROM python as base
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ADD / /code/
+WORKDIR /code
+
+ENV PATH "/code:/root:/root/.poetry/bin:${PATH}"
+
+# add and install python requirements
+RUN poetry install
```

Comment:
Currently, this being after the `ADD` (and since we're not using a volume) means that any code change has to go through this dependency-installation step, which can be fairly slow. One way to avoid that would be to reorder this slightly as below, so that the installation step can be cached for most code changes, and only re-runs if the dependencies have changed
```
COPY pyproject.toml poetry.lock /code
RUN poetry install
COPY / /code
```

Reply:
Made the change